app/src/main/res/xml/preferences_appearance.xml

```diff
@@ -48,5 +48,10 @@
         android:entries="@array/text_enter_labels"
         android:entryValues="@array/text_enter_values"
         android:defaultValue="@string/pref_default_text_enter"/>
+    <CheckBoxPreference
+        android:defaultValue="false"
```

Comment:
Same as above: add the default value to `res/values/config.xml` (see other preferences) and use that instead of the hard-coded `false`.